spec.html

```diff
@@ -6241,20 +6254,36 @@
     </emu-clause>
   </emu-clause>
 
+  <emu-clause id="sec-continents">
+    <h1>Continents</h1>
+
+    <p>A <dfn id="continent">continent</dfn> is an independent unit of ECMAScript computation, and is the largest unit that this specification deals with. Everything within this specification is assumed to occur within a single continent.</p>
+
+    <p>The host environment must create a continent prior to the execution of any Jobs or the evaluation of any ECMAScript code.</p>
+
+    <p>A continent may contain multiple realms, as defined in <emu-xref href="#sec-code-realms"></emu-xref>. Each realm belongs to exactly one continent. Depending on the host environment, code executed within a given continent's realms may or may not be able to reach objects from other realms in that continent. However, code executed in one continent must not be able to directly reference or share objects from another continent.</p>
+
+    <p>Each continent contains a set of job queues, as defined in <emu-xref href="#sec-jobs-and-job-queues"></emu-xref>.</p>
+
+    <emu-note>Typically, a continent is implemented as a single thread of execution. For example, in a web browser, each unit of related similar-origin browsing contexts consists of a single continent, as does each worker.</emu-note>
```

Comment:
I'm trying to provide a somewhat-intuitive conceptual understanding for authors and implementers as to what a continent corresponds to in the "real world." Before embarking on this journey, it wasn't clear to me at all whether we were supposed to spin up a separate instance of the ES spec for each global, or each browser, or each browser window, or what. I am trying to point out that it's essentially one continent (or "instance of the ES spec") per thread/event loop. The "For example" tries to bring that even closer to real-world experience, showing a system where many realms can coexist (with "political barriers" between them, cf. [es-discuss](https://esdiscuss.org/topic/holy-mixed-metaphors-batman-was-event-loops-in-navigated-away-from-windows#content-2)) within the same continent.